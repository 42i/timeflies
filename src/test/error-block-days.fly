leave 2012-02-05 2012-02-12 Jake
sick 2012-02-13 2012-02-17 Elwood
day 2012-02-18 8 13
off aaa
leave 2012-02-05 2012-02-12 Bilbo
sick 2012-02-13 2012-02-17 Gollum
