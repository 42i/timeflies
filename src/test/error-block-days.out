error-block-days.fly:1: ERROR : no current day for instruction "leave 2012-02-05 2012-02-12 Jake".
error-block-days.fly:2: ERROR : no current day for instruction "sick 2012-02-13 2012-02-17 Elwood".
error-block-days.fly:4: ERROR : bad time "aaa" in instruction "off aaa".
error-block-days.fly:5: ERROR : bad time "2012-02-05" in instruction "leave 2012-02-05 2012-02-12 Bilbo".
error-block-days.fly:6: ERROR : bad time "2012-02-13" in instruction "sick 2012-02-13 2012-02-17 Gollum".
5 errors.
